SOARback/initDB.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+interventions_dbFile=db/SOARback_interventions.db
+users_dbFile=db/SOARback_users.db
+
+if test -f "$interventions_dbFile"; then
+    echo "SOARback interventions db exists at $interventions_dbFile"
+else
+    echo "SOARback interventions db does not already exist, creating at $interventions_dbFile"
+    sqlite3 $interventions_dbFile "
```

Comment:
To my knowledge sqlite3 is a server less way to store the data. It is also already in our library?